api/views.py

```diff
@@ -266,16 +267,17 @@ def _only_latest_date(self, iterable):
             prev_ = next_
         yield next_
 
-    def listify(self, iterable, request, format=None):
+    def listify(self, iterable, request=None, format=None):
         """Some renderers require lists instead of any iterables for rendering.
         This function conditionally turns iterables into lists based on the
         format requested by browser."""
         # choose either '?format=...' or '/url.format' or None
-        format_ = (request.query_params.get('format') or format or '').lower()
+        if request is not None:
+            format_ = (request.query_params.get('format') or format or '').lower()
 
-        if format_ in self.formats_requiring_lists:
-            # list-ify the generator for renderers requiring lists
-            return list(iterable)
+            if format_ in self.formats_requiring_lists:
+                # list-ify the generator for renderers requiring lists
+                return list(iterable)
```

Comment:
@pbanaszkiewicz , could we always listify, that is, always return `list(iterable)`?